### PR TITLE
Add Brutal v2 1800 PCB to QMK

### PR DESCRIPTION
**Pull request by awkannan**
## Description

Add the Brutal v2 1800 PCB to QMK

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by drashna — APPROVED** [review]

File: keyboards/cannonkeys/brutalv2_1800/readme.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Brutal v2 1800
+
+Brutal v2 1800 Keyboard
+
+* Keyboard Maintainer: [Andrew Kannan](https://github.com/awkannan)  
+* Hardware Supported: RP2040
```
Comment (fauxpark) [inline keyboards/cannonkeys/brutalv2_1800/readme.md:6]:
```suggestion
* Keyboard Maintainer: [Andrew Kannan](https://github.com/awkannan)
* Hardware Supported: RP2040
```

File: keyboards/cannonkeys/brutalv2_1800/keymaps/via/keymap.c [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    _BASE,
+    _FN1,
+    _FN2,
+};
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+	[_BASE] = LAYOUT_all(
+        KC_ESC, KC_F1, KC_F2, KC_F3, KC_F4, KC_F5, KC_F6, KC_F7, KC_F8, KC_F9, KC_F10, KC_F11, KC_F12, KC_PSCR, KC_DEL, KC_INS, KC_PGUP, KC_PGDN,
+		KC_GRV, KC_1, KC_2, KC_3, KC_4, KC_5, KC_6, KC_7, KC_8, KC_9, KC_0, KC_MINS, KC_EQL, KC_BSPC, KC_BSPC, KC_NLCK, KC_PSLS, KC_PAST, KC_PMNS,
```
Comment (fauxpark) [inline keyboards/cannonkeys/brutalv2_1800/keymaps/via/keymap.c:34]:
```suggestion
		KC_GRV, KC_1, KC_2, KC_3, KC_4, KC_5, KC_6, KC_7, KC_8, KC_9, KC_0, KC_MINS, KC_EQL, KC_BSPC, KC_BSPC, KC_NUM,  KC_PSLS, KC_PAST, KC_PMNS,
```

File: keyboards/cannonkeys/brutalv2_1800/keymaps/default/keymap.c [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+enum layer_names {
+    _BASE,
+    _FN1,
+};
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+	[_BASE] = LAYOUT_all(
+        KC_ESC, KC_F1, KC_F2, KC_F3, KC_F4, KC_F5, KC_F6, KC_F7, KC_F8, KC_F9, KC_F10, KC_F11, KC_F12, KC_PSCR, KC_DEL, KC_INS, KC_PGUP, KC_PGDN,
+		KC_GRV, KC_1, KC_2, KC_3, KC_4, KC_5, KC_6, KC_7, KC_8, KC_9, KC_0, KC_MINS, KC_EQL, KC_BSPC, KC_BSPC, KC_NLCK, KC_PSLS, KC_PAST, KC_PMNS,
```
Comment (fauxpark) [inline keyboards/cannonkeys/brutalv2_1800/keymaps/default/keymap.c:33]:
```suggestion
		KC_GRV, KC_1, KC_2, KC_3, KC_4, KC_5, KC_6, KC_7, KC_8, KC_9, KC_0, KC_MINS, KC_EQL, KC_BSPC, KC_BSPC, KC_NUM,  KC_PSLS, KC_PAST, KC_PMNS,
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Comment by awkannan** [comment]
@faxupark Ready for re-review :) Thank you!

**Review by fauxpark — APPROVED** [review]

**Pull request merged**